piptools/scripts/compile.py

```diff
@@ -320,26 +344,26 @@ def cli(
         elif is_setup_file:
             setup_file_found = True
             try:
-                metadata = project_wheel_metadata(
-                    os.path.dirname(os.path.abspath(src_file)),
+                metadata = build_project_metadata(
+                    src_file=src_file,
+                    build_distributions=build_deps_for_distributions,
                     isolated=build_isolation,
+                    quiet=log.verbosity < 0,
```

Comment:
```suggestion
                    quiet=log.verbosity <= 0,
```

Addressees https://github.com/jazzband/pip-tools/pull/1681/files#r1299451803.

Reply:
Will fix

Reply:
Done